xray/commands/audit/generic/auditmanager.go

```diff
@@ -274,25 +282,29 @@ func getTechDependencyTree(params *Params, tech coreutils.Technology) (dependenc
 		e = errorutils.CheckError(fmt.Errorf("%s is currently not supported", string(tech)))
 	}
 
-	return dependencyTrees, e
+	params.dependencyTrees = dependencyTrees
+	return services.FlattenGraph(dependencyTrees), e
```

Comment:
Should we actually flatten the graph, even if e is not nil?
Let's also rename `e` to `err` throughout this function.